docs/quick-start.md

```diff
@@ -0,0 +1,220 @@
+# gRPC Swift Quick Start
+
+## Before you begin
+
+### Prerequisites
+
+#### Swift Version
+
+gRPC requires Swift 5.0 or higher.
+
+#### Install Protocol Buffers v3
+
+Install the protoc compiler that is used to generate gRPC service code. The
+simplest way to do this is to download pre-compiled binaries for your
+platform (`protoc-<version>-<platform>.zip`) from here:
+[https://github.com/google/protobuf/releases][protobuf-releases].
+
+* Unzip this file.
+* Update the environment variable `PATH` to include the path to the `protoc`
+  binary file.
+
+### Download the example
+
+You'll need a local copy of the example code to work through this quickstart.
+Download the example code from our GitHub repository (the following command
+clones the entire repository, but you just need the examples for this quickstart
+and other tutorials):
+
+```sh
+$ # Clone the repository at the latest release to get the example code:
+$ git clone -b 1.0.0-alpha.6 https://github.com/grpc/grpc-swift
```

Comment:
Should we really mention a specific tag here? How about just cloning the `nio` branch for now?

Reply:
I don't mind: I just went for consistency with other quick start guides (and the README).

Reply:
Thanks for the info. Either is fine with me then.